src/script/cpp_api/s_env.cpp

```diff
@@ -79,6 +86,13 @@ class LuaABM : public ActiveBlockModifier {
 	{
 		return m_max_y;
 	}
+	virtual void change(float interval, u32 chance, s16 min_y, s16 max_y)
+	{
+		m_trigger_interval = interval;
+		m_trigger_chance = chance;
+		m_min_y = min_y;
+		m_max_y = max_y;
+	}
```

Comment:
I wonder why we have these "java-like" getters and setters anyway. would be easier to make the members public.